pkgs/development/libraries/gdal/default.nix

```diff
@@ -86,6 +86,41 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  doInstallCheck = true;
+  # preCheck rather than preInstallCheck because this is what pytestCheckHook
+  # calls (coming from the python world)
+  preCheck = ''
+    pushd ../autotest
+    # something has made files here read-only by this point
+    chmod -R u+w .
+
+    export HOME=$(mktemp -d)
+    export PYTHONPATH="$out/${pythonPackages.python.sitePackages}:$PYTHONPATH"
+  '';
+  installCheckInputs = with pythonPackages; [
```

Comment:
Is that still python2?

Reply:
No. Looks like it gets overridden when this is included in a `pythonPackages`.

It's not something I've changed anyway, and not something I particularly want to go screwing with.